src/tetris/field/remoteField.ts

```diff
@@ -75,5 +100,41 @@ export default class RemoteField {
 			}
 		}
 	}
+
+	private _setupText(): void {
```

Comment:
`_setupText` is very generic... perhaps you can find a better name for this method.